src/shared/ScreenLayout.js

```diff
@@ -58,8 +60,15 @@ const ScreenLayout = (props) => {
         </TitleContainer>
         {children}
       </BodyContainer>
+
       <Footer>
-        <FooterButton title={btnLabel} onPressHandler={onPressHandler} />
+        {showFooter ? (
```

Comment:
what if instead of making a new variable for `showFooter` you just check `btnLabel`? 
Then in Lesson.js you do `btnLabel={completedGrammars[grammar.id] ? btnLabel : null}`